rpcd/playbooks/elasticsearch.yml

```diff
@@ -54,6 +56,27 @@
       tags:
         - elasticsearch-pre-install
   roles:
-    - "elasticsearch"
+    - role: "ansible-elasticsearch"
+      es_instance_name: "openstack"
```

Comment:
That has nothing to do with group vars, and everything to do with the tags. The artitact prep should *only* do the package installs, and because that role is not set to use the tags we do, that's not all that's going on in the artifact build for this role. So you need to ensure that *only* the package install happens with absolutely no configuration of any kind.